test/test_torch.py

```diff
@@ -1779,8 +1779,7 @@ def test_deterministic_cumsum(self, device):
             self.assertEqual(res0, res_cpu, atol=1e-3, rtol=1e-2)
 
     @onlyCUDA
-    @largeTensorTest('24GB', device='cuda')
-    @largeTensorTest('24GB', device='cpu')
+    @largeTensorTest('49GB')
```

Comment:
Note to reviewers: the original value was clearly incorrect, if you look at the code. `b` and `d` are both slightly more than 24GB, and are active at the same time in the function.